static-checks/rule-identifiers/test.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+
+import re
+from collections import defaultdict
+
+from lib import util, results, oscap, versions
+
+
+profile_references = {
```

Comment:
Consider moving this to a separate file, so it's easier to navigate/edit for developers (potentially). But it's not a big deal, keeping it in the test might keep the information fragmentation low.

Reply:
I prefer having it here as you mentioned to keep information fragmentation low.